hw/top_earlgrey/ip/pinmux/data/autogen/pinmux.hjson

```diff
@@ -41,8 +78,56 @@
       default: "32",
       local: "true"
     },
+    { name: "NDioPads",
+      desc: "Number of dedicated IO pads",
+      type: "int",
+      default: "14",
+      local: "true"
+    },
+    { name: "NWkupDetect",
+      desc: "Number of wakeup detectors",
+      type: "int",
+      default: "8",
+      local: "true"
+    },
+    { name: "WkupCntWidth",
+      desc: "Number of wakeup counter bits",
+      type: "int",
+      default: "8",
+      local: "true"
+    },
+    // If a bit is set to 0 in this vector, state retention
+    // registers are activated when going to sleep
```

Comment:
instead of saying `state retention`, because that sounds like just the old value being latched, should we saying `activates low power behavior` per a certain register?

Reply:
good point, I'll change that.